crates/sui-graphql-e2e-tests/tests/stable/consistency/epochs/checkpoints.exp

```diff
@@ -225,49 +230,25 @@ Response: {
     "epoch_0": {
       "epochId": 0,
       "checkpoints": {
-        "nodes": [
-          {
-            "sequenceNumber": 1
-          },
-          {
-            "sequenceNumber": 2
-          },
-          {
-            "sequenceNumber": 3
-          }
-        ]
```

Comment:
Same here, very confused as to why this changed. Maybe it has something to do with the trait impl for RawQuery and Cursor?